### ISSUE TITLE
missing rotten tomatoes audience score

### ISSUE THREAD

**Issue by michpas** [ilarramendi/Cover-Ratings#32]
Allmost movies have rotten tomatoes audience score, but this one not.

![image](https://user-images.githubusercontent.com/66222810/117585673-f3efb500-b113-11eb-89be-4d658bea70fa.png)

The folder date is right, title is right and RT has the information. https://www.rottentomatoes.com/m/1917_2019

What could be the reason that this is missing?

**Comment by michpas** [comment]
Bit similar issue for:
![image](https://user-images.githubusercontent.com/66222810/117585752-58127900-b114-11eb-893a-2f8fa40a0ddb.png)
Plex shows 91% and 60% (see image).
Your cover script is not showing the RT ratings.

https://www.rottentomatoes.com/m/pg_psycho_goreman

**Comment by ilarramendi** [comment]
Can you check on the db file that the ratings are missing from there too?

**Comment by ilarramendi** [comment]
Also is 1917 movie `'RTURL': '/m/1917_2019'` ?
The same for psyco `'RTURL': '/m/pg_psycho_goreman'`

**Comment by michpas** [comment]
Hi. On 1917:

              "path": "/media/1917 (2019)",
              "title": "1917",
              "year": "2019",
              "seasons": {},
              "type": "movie",
              "mediainfoDate": "09/05/2021",
              "mediainfo": {
                     "color": "SDR",
                     "resolution": "HD",
                     "codec": "AVC",
                     "source": "",
                     "languages": [
                            "FRE",
                            "ENG"
                     ]
              },
              "metadataDate": "09/05/2021",
              "ratings": {
                     "TMDB": {
                            "icon": "TMDB",
                            "value": "7.9"
                     },
                     "MTC": {
                            "icon": "MTC",
                            "value": "7.8"
                     },
                     "IMDB": {
                            "icon": "IMDB",
                            "value": "8.3"
                     },
                     "RT": {
                            "icon": "RT",
                            "value": "8.9"
                     },
                     "LB": {
                            "icon": "LB",
                            "value": "8.2"
                     }
              },
              "ageRating": "R",
              "certifications": [],
              "cover": "https://image.tmdb.org/t/p/original/iZf0KyrE25z1sage4SYFLCCrMi9.jpg",
              "backdrop": "https://image.tmdb.org/t/p/original/2lBOQK06tltt8SQaswgb8d657Mv.jpg",
              "LBURL": "https://letterboxd.com/csi/film/1917/rating-histogram/"
       },

It seems for some reason the RTURL" is not there

**Comment by michpas** [comment]
On psyco: also missing the RTURL link:

**      "/media/Psycho Goreman (2021)": {
              "ids": {
                     "IMDBID": "tt11252440",
                     "TMDBID": "tt11252440"
              },
              "path": "/media/Psycho Goreman (2021)",
              "title": "Psycho Goreman",
              "year": "2021",
              "seasons": {},
              "type": "movie",
              "mediainfoDate": "09/05/2021",
              "mediainfo": {
                     "color": "SDR",
                     "resolution": "HD",
                     "codec": "AVC",
                     "source": "",
                     "languages": [
                            "ENG",
                            "ENG"
                     ]
              },
              "metadataDate": "09/05/2021",
              "ratings": {
                     "TMDB": {
                            "icon": "TMDB",
                            "value": "6.3"
                     },
                     "MTC": {
                            "icon": "MTC",
                            "value": "6.9"
                     },
                     "IMDB": {
                            "icon": "IMDB",
                            "value": "6.2"
                     }
              },
              "ageRating": "NR",
              "certifications": [],
              "cover": "https://image.tmdb.org/t/p/original/foQnDv051kQ7satMQkbkPsUxzkD.jpg",
              "backdrop": "https://image.tmdb.org/t/p/original/fJI5Cqok6ItnLfccuzHtXk5QQg2.jpg"
       },**

**Comment by ilarramendi** [comment]
The problem with 1917 that the RT server for some reason returns:
```
{
               "name": "1917",
               "year": 2020,
               "url": "/m/1917_2019",
               "image": "https://resizing.flixster.com/9vcARH2aEDl7YNykHrfFkjKXKFg=/fit-in/80x80/v1.bTsxMzIwMzIxOTtqOzE4ODI3OzEyMDA7Mjc2NTs0MDk2",
               "meterClass": "certified_fresh",
               "meterScore": 89,
               "castItems": [
                    {
                         "name": "George MacKay",
                         "url": "/celebrity/george-mackay"
                    },
                    {
                         "name": "Dean-Charles Chapman",
                         "url": "/celebrity/dean_charles_chapman_2"
                    },
                    {
                         "name": "Mark Strong (II)",
                         "url": "/celebrity/mark_strong"
                    }
               ],
               "subline": "George MacKay, Dean-Charles Chapman, Mark Strong (II), "
          }
```

In the url says 1917_2019 but the year says 2020,  thats realy strange.
The problem with Psycho Goreman is that you have the movie stored as: `Psycho Goreman`
and in RT is `PS: Psycho Goreman`  
I have no idea how plex gets the results (the probably have a RT api key and cache that on their servers, so you just get ratings from their servers), ill have to investigate that.

**Comment by michpas** [comment]
> On psyco: also missing the RTURL link:
> 
> ** "/media/Psycho Goreman (2021)": {
> "ids": {
> "IMDBID": "tt11252440",
> "TMDBID": "tt11252440"
> },
> "path": "/media/Psycho Goreman (2021)",
> "title": "Psycho Goreman",
> "year": "2021",
> "seasons": {},
> "type": "movie",
> "mediainfoDate": "09/05/2021",
> "mediainfo": {
> "color": "SDR",
> "resolution": "HD",
> "codec": "AVC",
> "source": "",
> "languages": [
> "ENG",
> "ENG"
> ]
> },
> "metadataDate": "09/05/2021",
> "ratings": {
> "TMDB": {
> "icon": "TMDB",
> "value": "6.3"
> },
> "MTC": {
> "icon": "MTC",
> "value": "6.9"
> },
> "IMDB": {
> "icon": "IMDB",
> "value": "6.2"
> }
> },
> "ageRating": "NR",
> "certifications": [],
> "cover": "https://image.tmdb.org/t/p/original/foQnDv051kQ7satMQkbkPsUxzkD.jpg",
> "backdrop": "https://image.tmdb.org/t/p/original/fJI5Cqok6ItnLfccuzHtXk5QQg2.jpg"
> },**

On this one. I just noticed the imdb and tmbd stored information is the same. This is not correct.
The nfo file contains:
  <imdbid>tt11252440</imdbid>
  <tmdbid>670428</tmdbid>

Maybe something goes wrong with storing these ID's in the database file?

**Comment by ilarramendi** [comment]
Nice find! The problem with the ids should be fixed on v0.9, but the problem with RT is still happening, this will be a problem until i find a way to search by id on RT.
